atd-toolbox/README.md

```diff
@@ -2,6 +2,32 @@
 
 This is a collection of scripts which have been written to serve a one-time or infrequent purpose. They are not intended to be run in an automated fashion and commonly would be used from the command line when needed. They will each have different parameters and possible environment variables which need to be set for the script to function as intended.
 
+## Contents
+
+### Bulk user management - `bulk_user_management`
+
+Helper script to disable or enable all VZ users through the Auth0 API.
+
+### Data Model
+
+Various scripts used as part of launching the new data model for VZ v2.0. We want to keep these around for the few months after launch (August 2024) but they can be archived in the near future.
+
+### Engineering Areas and council districts load tools - `engineering_areas_load`, `load_council_districts`
```

Comment:
thanks for adding these script descriptions :)) this README got forgotten about with time methinks